phantom.json

```diff
@@ -1211,8 +1211,7 @@
                 "run_automation": {
                     "description": "Run automation on newly created artifact(s) (default: true)",
```

Comment:
Now that we remove `"default": true`, should we update this description as well?

Reply:
I have made the suggested code changes.